index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/>
+    <title>Nuweh website</title>
+  </head>
+  <body>
+    <header class="introduction-section">
+    <nav>
+      <div class="hamburger-icon d-md-none">
+        <i
+          class="fa fa-bars hamburger"
+          aria-hidden="true"
+          id="hamburger-icon"
+          onclick="toggleMobileMenu()"
+        ></i>
+        <div
+          class="dropdown-menu-nuweh hidden-dropdown-menu"
+          id="nav-dropdown-menu"
+        >
+          <i
+            class="fa-regular fa-x dropdown-x-icon"
+            id="nav-dropdown-x-icon"
+            onclick="toggleMobileMenu()"
+          ></i>
+          <ul class="nav-dropdown">
+            <li class="nav-dropdown-item">
+              <a
+                href="about.html"
+                class="nav-dropdown-item-link"
+                onclick="openDropdownItem(this,'#introduction-section')"
+                >About</a
+              >
+            </li>
```

Comment:
- [ ] Please make sure the User is able to navigate from the about page back to the home page in mobile menu, currently you can go from home mage to about page in the mobile version but not the other way around, I have recorded a demo for that please see :point_down: 

https://user-images.githubusercontent.com/97521557/217492598-fd25a343-e0d5-40f7-93c2-315ef88eca5a.mp4